scripts/tf_cnn_benchmarks/tf_cnn_benchmarks.py

```diff
@@ -1230,11 +1230,10 @@ def add_forward_pass_and_gradients(
         labels = host_labels
 
     with tf.device(self.devices[device_num]):
-      # Rescale to [0, 1)
-      images *= 1. / 256
-      # Rescale to [-1,1] instead of [0, 1)
-      images = tf.subtract(images, 0.5)
-      images = tf.multiply(images, 2.0)
+      # Rescale from [0, 255] to [0, 2]
+      images = tf.div(images, 127.5)
```

Comment:
Multiplicaiton is faster than division, so replace this line with `images *= 1./127.5`. Note the division `1./127.5` is done in Python code and only evaluated once instead of per image.